src/App.test.js

```diff
@@ -639,6 +640,54 @@ describe('App component', () => {
     });
   });
 
+  describe('zoho', () => {
+    it('Login view should not have zoho script', async () => {
```

Comment:
How about something more readable like this?
```suggestion
    it('should not have zoho script in login', async () => {
```